presto-client/src/main/java/com/facebook/presto/client/QueryResults.java

```diff
@@ -312,6 +314,8 @@ private static Object fixValue(TypeSignature signature, Object value)
             case INTERVAL_DAY_TO_SECOND:
             case DECIMAL:
                 return String.class.cast(value);
+            case CHAR:
+                return Strings.padEnd(String.class.cast(value), getCharTypeLength(signature), ' ');
```

Comment:
according to SQL standard, `CHAR(x)` always has `x` characters. Internally `CHAR(x)` is represented as variable length slice. However, when presenting to client it has to be padded with spaces to have length of `x`. We don't want to transmit fixed length `CHARs`, because spaces would waste bandwidth.